apps/alerts/lib/cache/store.ex

```diff
@@ -121,6 +121,7 @@ defmodule Alerts.Cache.Store do
         read_concurrency: true
       ])
 
+    # no cover
```

Comment:
question (non-blocking): For my own curiosity, what does this do?

Reply:
I believe this does nothing other than let the developer know that the return is deliberately ignored/not covered.  I added it just so it matched all the other calls in the function.